src/main/java/com/twelfthmile/yuga/YugaMethods.java

```diff
@@ -24,7 +24,7 @@ static int lookAheadForNum(String str, int index) {
             c = str.charAt(i);
             if (c == Constants.CH_SPACE) {
             }
-            else if (Util.isNumber(c))
+            else if (Util.isNumber(c) && !Util.isAlpha(str.charAt(i + 1)))
```

Comment:
Just because I got the notification

This will throw an exception if `i = (str.length() - 1)`, if you have a couple of spaces and then a number, which is at the end of a message. Check if `i + 1` exists, first

Reply:
Changed the logic.